.github/workflows/test.yml

```
@@ -0,0 +1,251 @@
+      if: always()
+      with:
+        name: playwright-report
+        path: playwright-report/
+        retention-days: 30
+
+  security-audit:
+    name: Security Audit
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)